src/course-unit/CourseUnit.jsx

```diff
@@ -139,6 +140,24 @@ const CourseUnit = ({ courseId }) => {
               />
             ) : null}
           </TransitionReplace>
+          {courseUnit.upstreamInfo.upstreamLink && (
+            <AlertMessage
+              title={intl.formatMessage(
+                messages.alertLibraryUnitReadOnlyText,
+                {
+                  link: (
+                    <Alert.Link
+                      className="ml-1"
+                      href={courseUnit.upstreamInfo.upstreamLink}
+                    >
```

Comment:
Nit/optional: is it possible to make this a React-router link, and just use the `upstreamRef` to figure out the URL rather than trusting the backend? Maybe more trouble than it's worth. Fine to leave it as is.